src/index.html

```diff
@@ -11,7 +11,203 @@
+      class="form"
+      method="post"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label for="surname">Surname:</label>
+
+        <input
+          id="surname"
+          type="text"
+          name="surname"
+          autocomplete="off"
+          class="input"
+        >
+
+        <br>
+
+        <label for="name">Name:</label>
+
+        <input
+          id="name"
+          type="text"
+          name="name"
+          autocomplete="off"
+          class="input"
+        >
+
+        <br>
+
+        <label for="age">How old are You?</label>
+
+        <input
+          id="age"
+          type="number"
+          name="age"
+          value="12"
+          class="input"
+          min="1"
+          max="100"
+        >
+
+        <br>
+
+        <label for="date">Full date of birth:</label>
+
+        <input
+          id="date"
+          type="date"
+          name="dateOfBirth"
+          value="mm/dd/yyyy"
+          class="input"
+        >
+
+        <br>
+
+        <label for="checkbox">I accept the term of the agreement</label>
+
+        <input
+          id="checkbox"
+          type="checkbox"
+          name="terms"
+        >
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label for="email">E-mail:</label>
+
+        <input
+          id="email"
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          class="input"
+          required
+        >
+
+        <br>
+
+        <label for="password">Password:</label>
+
+        <input
+          id="password"
+          type="password"
+          name="password"
+          minlength="8"
+          maxlength="30"
+          required
+        >
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label>Do you love cats?</label>
+
+        <input
+          id="yes"
+          type="radio"
+          name="cat"
+          value="yes"
+          class="input"
+        >
+
+        <label for="yes" class="input">Yes</label>
+
+        <input
+          id="no"
+          type="radio"
+          name="cat"
+          value="No"
+          class="input"
+        >
+
+        <label for="no" class="input">No</label>
+
+        <br>
+
+        <label for="color">What is your favorite color?
+          <input
+            id="color"
+            type="color"
+            name="color"
+            class="input"
+          >
+        </label>
+
+        <br>
+
+        <label for="time">What time do you go to bed?</label>
+
+        <input
+          id="time"
+          type="time"
+          name="timeToBed"
+          step="2"
+          class="input"
+        >
+
+        <br>
+
+        <label for="car" class="input">
+          What are your favorite brand of cars?
```

Comment:
Close label tag here:
```suggestion
        <label for="car" class="input">
          What are your favorite brand of cars?
        </label>
```